src/components/ProcessCreate/CreationProgress.tsx

```diff
@@ -43,24 +43,28 @@ export const CreationProgress = ({ error, sending, step }: CreationProgressProps
   }, [step])
 
   return (
-    <Stack direction='column' gap={4}>
-      <Text>{t('process_create.creation_steps_description')}</Text>
-      <Heading as='h2' size='md' textAlign='center'>
-        {t('process_create.creation_steps_title')}
-      </Heading>
+    <Stack direction='column' gap={5} p={0}>
+      <Box bgImage='url(/assets/spreadsheet-confirm-modal.png)' height='150px' bgSize='cover' borderRadius='lg' />
```

Comment:
@gerouvi the asset url should be imported via imports, otherwise won't work if located in a different path than root (`/`).